client/my-sites/current-site/notice.jsx

```diff
@@ -138,11 +138,11 @@ class SiteNotice extends React.Component {
 		}
 		return (
 			<div className="site__notices">
+				{ this.freeToPaidPlanNotice() }
 				{ this.getSiteRedirectNotice( site ) }
 				<QuerySitePlans siteId={ site.ID } />
 				{ this.domainCreditNotice() }
 				{ this.jetpackPluginsSetupNotice() }
-				{ this.freeToPaidPlanNotice() }
 				<DomainToPaidPlanNotice />
```

Comment:
Should also move this one, as it uses `SidebarBanner`.

Reply:
Ah, didn't notice. I'll move it.